### PR TITLE
docs(rxSelectFilter): Page objects to jsdoc3

### PR DESCRIPTION
**Pull request by Droogans**
### LGTMs
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/components/rxSelectFilter/scripts/rxSelectFilter.page.js [hunk 1]
```diff
@@ -20,20 +21,29 @@ var rxSelectFilter = {
...
+     *             US: true
+     *         },
+     *         Status: {
+     *             All: false,
+     *             Open: true
+     *         }
+     *     });
+     *
+     *     expect(myPage.myTable.column('Account').data.then(_.uniq)).to.eventually.eql(['US']);
+     *     expect(myPage.myTable.column('Status').data.then(_.uniq)).to.eventually.eql(['Open']);
```
Comment (CITguy) [inline src/components/rxSelectFilter/scripts/rxSelectFilter.page.js]: Missing the closing `});` for the `it()`

**Comment by jj4th** [comment]
QE-LGTM - Pending addressing Ryan's comment

File: src/components/rxSelectFilter/scripts/rxSelectFilter.page.js [hunk 2]
```diff
@@ -20,20 +21,30 @@ var rxSelectFilter = {
...
+     * });
```
Comment (Droogans) [inline src/components/rxSelectFilter/scripts/rxSelectFilter.page.js]: @CITguy I got it down here, but your comment was above it so it's not an outdated diff.

Comment (CITguy) [inline src/components/rxSelectFilter/scripts/rxSelectFilter.page.js]: :+1:

**Comment by CITguy** [comment]
LGTM

**Pull request merged**